slack-lint/src/test/java/slack/lint/text/SpanPointMarkDangerousCheckDetectorTest.kt

```diff
@@ -0,0 +1,180 @@
+/*
+ * Copyright (C) 2021 Slack Technologies, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package slack.lint.text
+
+import org.intellij.lang.annotations.Language
+import org.junit.Test
+import slack.lint.BaseSlackLintTest
+
+/**
+ * Tests [SpanPointMarkDangerousCheckDetector].
+ */
+class SpanPointMarkDangerousCheckDetectorTest : BaseSlackLintTest() {
```

Comment:
I worry a bit about the qualified reference resolution (it's _very_ tricky to get right in Anvil), so I'd want to see a test or two that exercises different levels of imports (constant directly, class + reference, fully qualified reference).

Reply:
👍 https://github.com/slackhq/slack-lints/pull/14/commits/37277059a6d53622e65de562e17f0fdb519908b4